utils/sysrepo-load-modules.sh

```diff
@@ -48,8 +41,8 @@ INSTALL_MODULE_CMD() {
 }
 
 UPDATE_MODULE() {
-	CMD="'$SYSREPOCTL' -U $MODDIR/$1  -s '$MODDIR -v2"
-    eval $CMD
+    CMD="'$SYSREPOCTL' -U $MODDIR/$1  -s '$MODDIR' -v2"
```

Comment:
This breaks relations with the original script.  Everything below source - is copied from netopeer2

Reply:
Yeah well update didn't work because of the missing closing ' after $MODDIR, so confd-rebuild never worked.  I think we should break with netopeer2 anyway.

Reply:
Strange, works for me.